spec/stack.js

```diff
@@ -13,10 +13,62 @@ describe('Stack', () => {
 
   context('push()', () => {
```

Comment:
I would use `describe` here instead of `context`. `context` is generally used when there is something specific about the environment the test is running in